test/dlc_tests/sanity/test_dlc_labels.py

```diff
@@ -42,4 +44,71 @@ def test_dlc_major_version_label(image, region):
 
     assert major_version, f"{image} has no LABEL named 'dlc_major_version'. Please insert label."
 
-    LOGGER.info(f"{image} has 'dlc_major_version' = {major_version}")
+    test_utils.LOGGER.info(f"{image} has 'dlc_major_version' = {major_version}")
+
+
+@pytest.mark.integration("dlc_major_version_label")
+@pytest.mark.model("N/A")
+def test_dlc_version_dockerfiles(image):
+    """
+    Test to make sure semantic versioning scheme in Dockerfiles is correct
+
+    :param image: <str> ECR image URI
+    """
+    dlc_dir = os.getcwd().split('/test/')[0]
+    job_type = test_utils.get_job_type_from_image(image)
+    framework, fw_version = test_utils.get_framework_and_version_from_tag(image)
+    processor = test_utils.get_processor_from_image_uri(image)
+
+    root_dir = os.path.join(dlc_dir, framework, job_type, 'docker', fw_version)
+
+    # Skip older FW versions that did not use this versioning scheme
+    references = {
+        "tensorflow2": "2.2.0",
+        "tensorflow1": "1.16.0",
+        "mxnet": "1.7.0",
+        "pytorch": "1.5.0"
+    }
+    if test_utils.is_tf1(image):
+        reference_fw = "tensorflow1"
+    elif test_utils.is_tf2(image):
+        reference_fw = "tensorflow2"
+    else:
+        reference_fw = framework
+    if processor != "eia" and fw_version < references[reference_fw]:
+        pytest.skip("Not enforcing new versioning scheme on old images")
```

Comment:
nit: perhaps worth logging the image URI in the skip notification

Reply:
We have semantic versioning on all the older images in this repository as well.